.circleci/config.yml

```diff
@@ -12,12 +12,43 @@ jobs:
         environment:
           POSTGRES_USER: root
           POSTGRES_DB: arlo-test
+    working_directory: ~/arlo
     steps:
       - checkout
+      - run: sudo chown -R circleci:circleci /usr/local/bin
+      - run: sudo chown -R circleci:circleci /usr/local/lib/python3.7/site-packages
+      - restore_cache:
+          keys:
+            - yarn-packages-v3-{{ .Branch }}-{{ checksum "yarn.lock" }}
+      - restore_cache:
+          keys:
+            - pip-packages-v4-{{ .Branch }}-{{ checksum "Pipfile.lock" }}
+            - pip-packages-v4-{{ .Branch }}-
+            - pip-packages-v4-
+      - restore_cache:
+          keys:
+            - npm-packages-v6-{{ .Branch }}-{{ checksum "package-lock.json" }}
       - run:
+          parallelism: 4
```

Comment:
is it possible to configure this at a higher level in the file, or does it have to be set on each command?

Reply:
My understanding is that if you use `parallelism > 1` for sequential steps, it runs those steps that many times. So yes, we could but from my testing that made it slower, as random stuff would cause one run to be way slower than the others.